docs/link-flow.md

```diff
@@ -19,3 +19,15 @@ limitations under the License.
 This flow is normally originated from another surface and allows the reader to link this publication's subscription to that surface. See [Subscriptions APIs](./core-apis.md).
 
 The link flow would normally be executed by a 3p surface to let a user to claim an existing subscription. However, SwG client provides `linkAccount` and `setOnLinkComplete` APIs for testing purposes.
+
+# SwG Link Save flow (Not launched)
+
+This flow is normally originated from the publisher and allows the reader to link publication's subscription to the reader's account.
+
+To start saving subscription link, provide a request token corresponding to the readers subscription:
+
+subscriptions.saveSubscription(saveSubscriptionRequest);
```

Comment:
Add code section for this. Also please expand what request looks like. E.g. `{token: 'THE TOKEN'}`.